setup.py

```diff
@@ -16,6 +16,7 @@
         "pytest-mock",  # For creating mocks
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
         "hypothesis-jsonschema==0.19.0",  # JSON Schema fuzzer extension
+        "ape-solidity",  # Required for compiling test contracts
```

Comment:
Is there a way to test this functionality with the built-in package compiler instead?

Reply:
- [ ] Hmm.. Maybe we don't! I can try that.

However, I think having hello-world contracts in the tests is worth considering, for future testing purposes.

Reply:
Easy Peasy! I removed all the stuff related to solidity in the tests and am just using the json compiler.